Yoast/wordpress-seo
# Add tests for author and hierarchy builder

## Context


* We want all of the code covered in unit tests.

## Summary


This PR can be summarized in the following changelog entry:

* Adds unit tests for the indexable author builder and the indexable hierarchy builder.

## Relevant technical choices:

* In the indexable author builder, I added a third test. This means that some of the exact same code appeared three times. Therefore, I moved it to a `setUp` function. 

## Test instructions

This PR can be tested by following these steps:

* See all the tests pass.
* See that code coverage in `indexable-author-builder.php` and `indexable-hierarchy-builder.php` is 100%.

## UI changes

## Documentation

## Quality assurance


Fixes #14887
